cpp/open3d/geometry/PointCloud.cpp

```diff
@@ -479,8 +479,9 @@ PointCloud::RemoveRadiusOutliers(size_t nb_points, double search_radius) const {
     for (int i = 0; i < int(points_.size()); i++) {
         std::vector<int> tmp_indices;
         std::vector<double> dist;
-        size_t nb_neighbors = kdtree.SearchRadius(points_[i], search_radius,
-                                                  tmp_indices, dist);
+        size_t nb_neighbors = kdtree.SearchHybrid(
```

Comment:
Hi, 
Do you see any performance improvement in your experiments?
I just realised, Hybrid search is `RadiusSearch` -> `KnnSearch`.
Therefore, for the `search_radius` parameter, it will first conduct RadiusSearch, and then use KnnSearch to filter out the result. Therefore Hybrid Search might be slower than Radius Search in this case.